src/integration/common/analytics.amp.js

```diff
@@ -1,9 +1,10 @@
 export default () => {
-  it('AMP analytics config is loaded', () => {
-    expect(
-      amp.document.querySelector(
-        'amp-analytics script[type="application/json"]',
-      ).textContent,
-    ).toMatch('https://logws1363.ati-host.net?');
+  describe('Analytics', () => {
+    it('ATI', () => {
+      expect(
+        document.querySelector('amp-analytics script[type="application/json"]')
+          .textContent,
+      ).toMatch('https://logws1363.ati-host.net?');
```

Comment:
Can we have a `toMatchSnapshot()` here instead - it outputs the contents of the ATI script which will be useful for testing on demand radio in future.